src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -104,6 +106,34 @@ public Drive(SwerveDriveMap map, DoubleSupplier xSpeed, DoubleSupplier ySpeed, D
                 () -> !isBlue,
                 this);
 
+        // Auto Stuff!!
+
+        AutoBuilder.configure(
+                () -> estimator.getEstimatedPosition(), // Robot pose supplier
+                this::setPose, // Method to reset odometry (will be called if your auto has a
+                // starting pose)
+                this::getSpeeds, // ChassisSpeeds supplier. MUST BE ROBOT RELATIVE
+                (speeds, feedforwards) -> move(speeds), // Method that will drive the robot given ROBOT RELATIVE
+                // ChassisSpeeds. Also optionally outputs individual module
+                // feedforwards
+                new PPLTVController(0.02), // PPLTVController is the built in path following controller for differential
+                // drive trains
+                getMap().config, // The robot configuration
+                () -> {
+                    // Boolean supplier that controls when the path will be mirrored for the red
+                    // alliance
+                    // This will flip the path being followed to the red side of the field.
+                    // THE ORIGIN WILL REMAIN ON THE BLUE SIDE
+
+                    var alliance = DriverStation.getAlliance();
+                    if (alliance.isPresent()) {
+                        return alliance.get() == DriverStation.Alliance.Red;
+                    }
+                    return false;
+                },
```

Comment:
This should just be 
```suggestion
                () -> !isBlue,
```

Which should be functionally the same.